h2o-r/h2o-package/R/kvstore.R

```diff
@@ -236,13 +239,40 @@ h2o.getModel <- function(model_id) {
       # Response column needs to be parsed
       if (name == "response_column")
         value <- value$column_name
-      allparams[[name]] <<- value
+        
+      if (all == TRUE) {
+        names(value) <- name
+        return(value)
+      }
+        
       # Store only user changed parameters into parameters
       # TODO: Should we use !isTrue(all.equal(param$default_value, param$actual_value)) instead?
-      if (is.null(param$default_value) || param$required || !identical(param$default_value, param$actual_value))
-        parameters[[name]] <<- value
+      if (is.null(param$default_value) || param$required || !identical(param$default_value, param$actual_value)){
+        names(value) <- name
+        return(value)
+      }
     }
-  })
+    return(NULL)
+  }
+    
+  # get name, value pairs
+  allparams_key_val = llply(json$parameters, fill_pairs, all=TRUE)
```

Comment:
it seems we do not depend on plyr :( so this will probably not be possible without adding a dependency
@michalkurka WDYT?

Reply:
good catch, thank you @honzasterba 

yes, the code needs to work without any additional dependencies

Reply:
so can I use the previous approach with <<- in this case?